firefox-ios/RustFxA/RustFirefoxAccounts.swift

```diff
@@ -162,10 +163,18 @@ open class RustFirefoxAccounts {
         }
 
         let type = UIDevice.current.userInterfaceIdiom == .pad ? DeviceType.tablet : DeviceType.mobile
+
+        var capabilities: [DeviceCapability] = [.sendTab]
+        // We don't have access to nimbus unfortunately, so we grab the key ourselves here
+        let closeTabsPref = prefs?.boolForKey(PrefsKeys.FeatureFlags.RemoteTabManagement)
```

Comment:
Issue / Question: Hmmm, will this do the right thing if the user hasn't flipped the secret setting on? That is, if the feature is enabled via the Nimbus variable, and the user hasn't ever opened secret settings, will the pref ever be set?

I wonder if this module can export a new option set, like:

```swift
public struct RustFirefoxAccountsFeatures: OptionSet {
    let rawValue: Int

    static let closeRemoteTabs = RustFirefoxAccountsFeatures(rawValue: 1 << 0)
}
```

Then have `RustFirefoxAccounts.startup()` take that option set, and, finally, have the Firefox for iOS `Client` code get the Nimbus feature variable, set up the option set, and pass it to `startup`.

Reply:
Great catch! I think your correct in that the nimbus flag won't be used (only the user-specified.

I love the OptionSet Idea! There are a few places startup (and reconfig) are called that makes it slightly hard as it made a spiral effect of trying to pull feature flags everywhere. However I think we only care about AppDelegate right now so I think it's in a much better state.